src/views/Search.vue

```diff
@@ -164,23 +166,29 @@ async function submit_form(this: any) {
     console.log('submitting', formData.value)
 
     // submit the POST request to Valis
+    axios.interceptors.request.use((request) => {
+      request.transformResponse = [data => data]
+      return request
+    })
+
```

Comment:
I do something similar in https://github.com/sdss/zora/blob/main/src/views/Target.vue#L176-L187

Reply:
Well, you'd think I would have read a bit more into the code 🥲